androidApp/src/main/java/com/mbta/tid/mbta_app/android/pages/NearbyTransitPage.kt

```diff
@@ -153,6 +149,10 @@ fun NearbyTransitPage(
                             }
 
                             LaunchedEffect(navRoute) {
+                                if (navBarVisible) {
```

Comment:
I noticed that when navigating between Nearby Transit & Stop Details, the bottom nav bar would flicker in and out. This seems to resolve the problem